### DISCUSSION TITLE
Cannot add the Expense

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1568] [Q&A] [answered]
Hi Team,
I cannot add the expense in the Expense tracker
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/1b4b4db3-a055-498f-858e-5dcb03bba131)

**Comment by sreedharm07** [comment]
Hi @omswetha 
ping your ips of frontend,mysql and backend

**Reply by omdevops99** [reply]
Hi @sreedharm07 
mysql 
public IP: 3.90.231.75
private IP : 172.31.40.38

Frontend
public IP: 54.198.125.12
private IP: 172.31.45.20

Backend
public IP: 54.198.47.156
private IP: 172.31.36.13

**Reply by sreedharm07** [reply]
Hi @omswetha 
i just restarted frontend and backend servers. then i try do add expenses
i dint seen any error, its adding perfectly
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/5a8b18c5-afda-4a8f-850d-9bb2c227b707)

**Reply by omdevops99** [reply]
Can you please let me know, How did you restarted ?

**Reply by omdevops99** [reply]
hello @sreedharm07

**Reply by sreedharm07** [reply, marked as answer]
Systemctl restart frontend 
Systemctl restart backend

**Reply by omdevops99** [reply]
Thanks @sreedharm07
